openfasoc/generators/cryo-gen/tools/cryo-gen.py

```diff
@@ -214,7 +217,7 @@
 
 time.sleep(2)
```

Comment:
@Jerry-lqx please remove all the time.sleep() lines. I think these are here because we used to have them in tempsense.